### PR TITLE
chore(infra): update dependabot

### PR DESCRIPTION
**Pull request by h1dden-da3m0n**
### Description

Saw that this repo was still using the old dependabot config, so I updated it :wink: 

(hope u excuse my slight change in pom formatting, I can remove it if necessary)

### Changes

* update dependabot config from v1 to v2
* slightly alter pom formatting

### Issues

* n/a

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -7,24 +7,30 @@
...
   <groupId>io.jenkins.plugins</groupId>
   <artifactId>theme-manager</artifactId>
   <version>${revision}${changelist}</version>
   <packaging>hpi</packaging>
+
   <properties>
     <revision>0.6</revision>
     <changelist>-SNAPSHOT</changelist>
     <jenkins.version>2.222.1</jenkins.version>
+    <bom.artifactId>2.222.x</bom.artifactId>
```
Comment (timja) [inline pom.xml:20]: Why put these here they are never used any where else?

Comment (h1dden-da3m0n) [reply pom.xml:20]: Hi timja,
no particular reason realy, kind of copied the format from the plugin template (and what is found within jenkinsci/bom/sample-plugin).
Want me to revert it?

Comment (timja) [reply pom.xml:20]: Yes please

Comment (h1dden-da3m0n) [reply pom.xml]: Done

**Review by timja — APPROVED** [review]

**Comment by h1dden-da3m0n** [comment]
Thank you for the merge :tada:

**Pull request merged**